docs/src/components/about/TeamPhotoMasonry.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Masonry from '@mui/lab/Masonry';
+
+const teamPhotos = [
+  {
+    img: '/static/blog/2023-chamonix-retreat/skiers.jpeg',
+    title: 'Snacks',
+  },
+  {
+    img: '/static/blog/2022-tenerife-retreat/group-photo.jpeg',
+    title: 'Fern',
+  },
+  {
+    img: '/static/blog/2023-chamonix-retreat/team-dinner.jpeg',
+    title: 'Mushrooms',
+  },
+  {
+    img: '/static/about/working-table-tenerife.jpeg',
+    title: 'Tower',
+  },
+  {
+    img: '/static/blog/2022-tenerife-retreat/scuba-gear.jpeg',
+    title: 'Tower',
+  },
+  {
+    img: '/static/blog/2022-tenerife-retreat/outdoor-focus-group.jpeg',
+    title: 'Tower',
+  },
+  {
+    img: '/static/about/working-table-portugal.jpeg',
+    title: 'Tower',
+  },
+  {
+    img: '/static/about/snow-tea.jpeg',
+    title: 'Tower',
+  },
+  {
+    img: '/static/about/portugal-sight-seeing.jpeg',
+    title: 'Tower',
+  },
+];
+
+export default function TeamPhotoMasonry() {
+  return (
+    <Masonry columns={3} spacing={2} sx={{ mt: 4, mb: 1, minHeight: 780 }}>
```

Comment:
Do we need to use Masonry here, a simple CSS grid (without the masonry feature would be sufficient), no?

Reply:
Just realized I ended up not using this idea ⎯ deleted! 😬  I started out playing with a Masonry first before going to the infinite scrolling gallery that's up on the page now!